javascript/javascript-core/functions-i/setting-default-values-with-short-circuiting.md

```diff
@@ -75,11 +75,14 @@ function title(t) {
 
 ## Practice
 
-"Short circuit" the following code:
+Complete both functions so `shortCirA` and `shortCirB` return `'John Doe'` and `'Jane Doe'` respectively on Undefined. If `name` is defined, then the function should just return the name:
```

Comment:
```suggestion
Complete both functions so `shortCirA` and `shortCirB` return `'John Doe'` and `'Jane Doe'`, respectively. 

If `name` is defined, then each function should return the name:
```

Reply:
It's unclear that John doe and Jane doe should be returned on undefined

Reply:
See my new version